main.go

```diff
@@ -229,58 +233,96 @@ func main() {
 	eventHandler := alice.New(authConstructor)
 	cutoffHandler := alice.New()
 
-	// set up the registerer
-	basicConfig := webhookClient.BasicConfig{
-		Timeout:         config.Webhook.Timeout,
-		RegistrationURL: config.Webhook.RegistrationURL,
-		Request: webhook.W{
-			Config: webhook.Config{
-				URL: config.Webhook.Request.WebhookConfig.URL,
+	var acquirer *acquire.RemoteBearerTokenAcquirer
+	var periodicRegisterer *webhookClient.PeriodicRegisterer
```

Comment:
Because you have to call `Stop()` for every periodicRegisterer, I would change this to a slice and then add to it while starting all the registerers and at the end you can cycle through and call stop for every one:
https://github.com/xmidt-org/caduceator/pull/10/files#diff-7ddfb3e035b42cd70649cc33393fe32cR373

Reply:
I'll add this change in